### PR TITLE
Collect from Other Data Sources index page.

### PR DESCRIPTION
**Pull request by Melvin-CnC**
Made changes as per the Open Source Docs Migration Plan

## Purpose of this pull request

This pull request (PR) for edits to the Collect from Other Data Sources index page.

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by Melvin-CnC** [comment]
@kimsauce I haven't included the "Kubernetes Fluentd plugin" in the index edits as it was marked as a "Deprecated Solution". Hope that's ok?

File: docs/send-data/collect-from-other-data-sources/index.md [hunk 1]
```diff
@@ -4,11 +4,173 @@ title: Collect from Other Data Sources
...
+    <div className="box smallbox24 card">
+      <div className="container">
+      <a href="/docs/send-data/collect-from-other-data-sources/sumo-logic-open-source-projects"><img src={useBaseUrl('img/icons/operations/collect.png')} alt="icon" width="40"/><h4>Sumo Logic Open Source Projects</h4></a>
+      <p>Learn about open-source solutions and resources for customers via GitHub.</p>
+      </div>
+    </div>
+    <div className="box smallbox25 card">
+      <div className="container">
+      <a href="/docs/send-data/collect-from-other-data-sources/vmware-airwatch-integration"><img src={useBaseUrl('img/icons/operations/collect.png')} alt="icon" width="40"/><h4>VMware AirWatch</h4></a>
+      <p>Provide visibility for monitoring enterprise mobility management in your deployment.</p>
```
Comment (kimsauce) [inline docs/send-data/collect-from-other-data-sources/index.md:161]:
```suggestion
      <p>Provides visibility for monitoring enterprise mobility management in your deployment.</p>
```

File: docs/send-data/collect-from-other-data-sources/index.md [hunk 2]
```diff
@@ -4,11 +4,173 @@ title: Collect from Other Data Sources
...
+    </div>
+    <div className="box smallbox20 card">
+      <div className="container">
+      <a href="/docs/send-data/collect-from-other-data-sources/collect-statsd-metrics"><img src={useBaseUrl('img/icons/operations/collect.png')} alt="icon" width="40"/><h4>StatsD</h4></a>
+      <p>Learn how to perform a StatsD implementation.</p>
+      </div>
+    </div>
+    <div className="box smallbox21 card">
+      <div className="container">
+      <a href="/docs/send-data/collect-from-other-data-sources/collect-ruby-on-rails-logs"><img src={useBaseUrl('img/icons/traces.png')} alt="icon" width="40"/><h4>Ruby on Rails</h4></a>
```
Comment (kimsauce) [inline docs/send-data/collect-from-other-data-sources/index.md:136]:
```suggestion
      <a href="/docs/send-data/collect-from-other-data-sources/collect-ruby-on-rails-logs"><img src={useBaseUrl('img/icons/operations/collect.png')} alt="icon" width="40"/><h4>Ruby on Rails</h4></a>
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Two minor fixes!

**Review by kimsauce — APPROVED** [review]

**Pull request merged**